app/.gitignore

```diff
@@ -1 +1,4 @@
-/build
\ No newline at end of file
+/build
+
+google-services.json
+/src/main/assets/RFC_model_ir9_opset19.onnx
```

Comment:
Good initiative to simplify the PR.
but I'll find a solution of how to push the file into git.